23.0.0.2 Release Blog Post (#2982)
OpenLiberty/blogs: posts/2023-03-07-23.0.0.2.adoc

---
layout: post
title: "Test database connections and set server stop time out in Open Liberty 23.0.0.2"
# Do NOT change the categories section
categories: blog
author_picture: https://avatars3.githubusercontent.com/mbroz2
author_github: https://github.com/mbroz2
seo-title: Test database connections and set server stop time out in Open Liberty 23.0.0.2 - OpenLiberty.io

> seo-description: Open Liberty 23.0.0.2 introduces a new capability in Admin Center's Server Config tool to test database connections.  The `server stop` command also receives a new capability to specify the maximum amount of time to wait for the server to stop via the `--timeout` option.  This release also provides many significant bug fixes, including one that addresses a CVE in Jakarta RESTful Web Services 3.0 feature.

Reviewer: ```suggestion
seo-description: Open Liberty 23.0.0.2 introduces a new capability to test database connections with the Admin Center Server Config tool.  The server stop command adds the the --timeout option to specify the maximum amount of time to wait for the server to stop.  This release also provides many significant bug fixes, including one that addresses a CVE in Jakarta RESTful Web Services 3.0 feature.
```

> blog_description: Open Liberty 23.0.0.2 introduces a new capability in Admin Center's Server Config tool to test database connections.  The `server stop` command also receives a new capability to specify the maximum amount of time to wait for the server to stop via the `--timeout` option.  This release also provides many significant bug fixes, including one that addresses a CVE in Jakarta RESTful Web Services 3.0 feature.

Reviewer: ```suggestion
blog_description: Open Liberty 23.0.0.2 introduces a new capability to test database connections with the Admin Center Server Config tool.  The server stop command adds the the --timeout option to specify the maximum amount of time to wait for the server to stop.  This release also provides many significant bug fixes, including one that addresses a CVE in Jakarta RESTful Web Services 3.0 feature.
```
